spec/tests/techniques/technique_rule.rb

```diff
@@ -8,13 +8,13 @@
 ruleName = $params['NAME']
 
 describe "Add a directive and a rule"  do
-
   $directive_id ||= 1
   for directiveFile in directiveFiles.split(",") do
     file = File.read(directiveFile)
     data = JSON.parse(file)
     technique = data["techniqueName"]
-    directiveName = data["displayName"] + " (" + index + "," + String($directive_id) + ")"
+    directiveName = data["displayName"]
```

Comment:
We now check if the directive exists, we no longer need a name identifier which is not so readable.